invenio_communities/members/services/service.py

```diff
@@ -234,10 +234,8 @@ def _invite_factory(self, identity, community, role, visible, member,
         # Add member entry
         if member['type'] == 'user':
             # Create request
-            # TODO: Fix me with a nicer role title.
-            title = _('Invitation to join "{community}" as {role}.').format(
+            title = _('Invitation to join "{community}".').format(
```

Comment:
I would remove the `.`, looks weird?
```suggestion
            title = _('Invitation to join "{community}"').format(
```